appimagebuilder/modules/setup/libraries_patcher.py

```diff
@@ -0,0 +1,73 @@
+#  Copyright  2022 Alexis Lopez Zubieta
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a
+#  copy of this software and associated documentation files (the "Software"),
+#  to deal in the Software without restriction, including without limitation the
+#  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+#  sell copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+import logging
+import pathlib
+
+from appimagebuilder.modules.setup.environment import Environment
+from appimagebuilder.utils.finder import Finder
+from appimagebuilder.utils.patchelf import PatchElf, PatchElfError
+
+
+class LibrariesPatcher:
+    def __init__(self, appdir: pathlib.Path, env: Environment, finder: Finder = None):
+        self.appdir = appdir
+        self.env = env
+        self.finder = finder or Finder(appdir)
+        self.patchelf = PatchElf()
+        self.logger = logging.getLogger("BinaryPatching")
+        self.runtime_compat_dir = self.appdir / "runtime/compat"
+
+    def patch_rpaths(self):
+        libraries = self.finder.find(
```

Comment:
When/if #190 is merged, this will need to be modified to check ``AppDir.runtime.preserve``.

Reply:
This was an experiment to replace LD_LIBRARY_PATHS but it didn't worked as expected. In some scenarios it's ignored and the resulting bundle doesn't work as expected. Therefore I'm going to stick with the existent method.